gui/storage/views.py

```diff
@@ -723,15 +723,26 @@ def zpool_scrub(request, vid):
         )
     if request.method == "POST":
         with client as c:
+            pool_scan = c.call('pool.query', [['id', '=', int(vid)]], {'get': True})['scan']
+            scrub_progress = pool_scan['function'] == 'SCRUB' and pool_scan['state'] == 'RUNNING'
             if request.POST["action"] == "start":
-                c.call('pool.scrub', vid, 'START')
-                return JsonResp(request, message=_("The scrub process has been started"))
+                if not scrub_progress:
+                    c.call('pool.scrub', vid, 'START')
+                    return JsonResp(request, message=_("The scrub process has been started"))
+                else:
+                    return JsonResp(request, message=_('Scrub is already in progress'))
```

Comment:
```suggestion
                    return JsonResp(request, message=_('The scrub is already in progress'))
```

Reply:
I'm assuming you don't want the `The` here because you're removing `The` in all the other suggestions?

Reply:
whoops, thanks - got caught by that pre-lunch rush :)